@Sudhanva-Nadiger could you please check the CI check:
```
[warn] src/views/ProjectApiKeys.vue
[warn] Code style issues found in the above file. Run Prettier to fix.
Error: Process completed with exit code 1.
```

--- a/front/src/views/ProjectApiKeys.vue
+++ b/front/src/views/ProjectApiKeys.vue
@@ -43,11 +43,18 @@
                         <v-btn icon @click="dialog = false"><v-icon>mdi-close</v-icon></v-btn>
                     </div>
                     <p v-if="form.action === 'delete'">
-                    Deleting the API key can result in some agents or applications using it no longer being able to write telemetry data to this
-                    project.
+                        Deleting the API key can result in some agents or applications using it no longer being able to write telemetry data to this
+                        project.
                     </p>
                     <div class="subtitle-1">Description</div>
-                    <v-text-field ref="descriptionField" v-model="form.description" outlined dense autofocus :readonly="form.action === 'delete'"></v-text-field>
+                    <v-text-field
+                        ref="descriptionField"
+                        v-model="form.description"
+                        outlined
+                        dense
+                        autofocus
+                        :readonly="form.action === 'delete'"
+                    ></v-text-field>
                     <v-alert v-if="error" color="red" icon="mdi-alert-octagon-outline" outlined text>
                         {{ error }}
                     </v-alert>

